modes/mu4e/evil-collection-mu4e.el

```diff
@@ -155,14 +155,14 @@ with older release versions of `mu4e.'"
      "o" mu4e-headers-change-sorting
      "j" mu4e-headers-next
      "k" mu4e-headers-prev
-     "gr" mu4e-headers-rerun-search
-     "b" mu4e-headers-search-bookmark
-     "B" mu4e-headers-search-bookmark-edit
+     "gr" mu4e-search-rerun
```

Comment:
Setting `revert-function-function` to `mu4e-search-return` in `mu4e-headers-mode` smells good, which let evil-collection maintain less keybindings. e.g. https://github.com/emacs-dashboard/emacs-dashboard/pull/352

If you would like to PR mu4e, 😀. Note, it's a bonus.

Reply:
I have saved it in my inbox. Once my PR to mu merged, I will open a new PR.